lib/ecto/adapters/postgres/connection.ex

```diff
@@ -67,6 +67,8 @@ if Code.ensure_loaded?(Postgrex) do
 
     @impl true
     def prepare_execute(conn, name, sql, params, opts) do
+      name = if opts[:prepare] == :named, do: name, else: ""
```

Comment:
I think we can remove this change. Given that only Postgres will pass `:unnamed` to `Ecto.Adapters.SQL.execute`, we can set the name to "" inside `Ecto.Adapters.SQL.execute`. WDYT?

Reply:
Makes sense. Updated :)